node/rpc/src/wallet_api.rs

```diff
@@ -43,10 +44,8 @@ where
             Ok(actor_balance.to_string())
         }
         Err(e) => {
-            if e == "Address not found" {
-                return Ok(BigUint::default().to_string());
-            }
-            Err(e.into())
+            warn!("Error retrieving actor state: {}", e);
+            Ok(BigUint::default().to_string())
```

Comment:
Can't comment above, but on the none case, a balance of 0 should be returned. (the ok_or above) since this seemed to be checking that way before for some reason